pages-e2e/features/_utils/copyWorkspaceAssets.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="../setup-types.d.ts" />
+
+import { cp, readdir } from 'fs/promises';
+import { join } from 'path';
+
+/**
+ * Copies all the files present in the `./assets` folder into the
+ * WORKSPACE_DIR folder (merging potentially already existing directories)
+ */
+export async function copyWorkspaceAssets(): Promise<void> {
```

Comment:
@GregBrimble I apply this to all my features, it's a more convenient way of copying things over without needing to care about file names etc...

I wonder if this could/should be baked in the framework itself 🤔 
(like we could have the convention that a folder named a certain way present in a feature just gets automatically copied into the fixture)